dlc-manager/src/sub_channel_manager.rs

```diff
@@ -1772,12 +1779,13 @@ where
 
         let (
             split_tx_adaptor_signature,
-            commitment_signed,
+            (commitment_signed, commit_tx_number),
```

Comment:
I guess this has to be the same number as the one that you pass in when calling `with_useable_channel_lock` just below.

Reply:
Mmm I guess it is the same but I don't know if there is a case where one party would maybe not yet have sent an RAA. That would probably mess things up though.

Reply:
Hmm, wouldn't it then fail when verifying the `commit_tx_number` provided by the caller?

Reply:
Within the LN protocol it is possible for parties to have a different commitment number at a given time. So the one provided by the caller could potentially be different than the one received here (since the one provided is the one for the local commitment tx while the received one is for the remote commitment tx). But I doubt that we can reach a state where the number differ without first encountering other issues.

Reply:
Thanks for clarifying!